File: Week 1 C/cash.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    // Get input
    int change_owed;
    do
    {
        change_owed = get_int("Change owed: ");
    }
    while (change_owed < 0);

    // 25
    int sum_25 = change_owed / 25;
    int remain_25 = change_owed - 25 * sum_25;

    // 10
    int sum_10 = remain_25 / 10;
    int remain_10 = remain_25 - 10 * sum_10;

    // 5
    int sum_5 = remain_10 / 5;
    int remain_5 = remain_10 - 5 * sum_5;

    // Sum up
    int sum = sum_25 + sum_10 + sum_5 + remain_5;
    printf("%i\n", sum);
}

File: Week 6 Python/cash.py
from cs50 import get_float

while True:
    n = get_float("Change owed: ") * 100
    if n >= 0:
        break

counter = 0

n_25 = n
while n_25 >= 25:
    n_25 -= 25
    counter += 1

n_10 = n_25
while n_10 >= 10:
    n_10 -= 10
    counter += 1

n_5 = n_10
while n_5 >= 5:
    n_5 -= 5
    counter += 1

n_1 = n_5
while n_1 >= 1:
    n_1 -= 1
    counter += 1

print(counter)
